compiler-rt/lib/tysan/tysan_interceptors.cpp

```diff
@@ -108,6 +108,14 @@ INTERCEPTOR(void *, malloc, uptr size) {
   return res;
 }
 
+#if SANITIZER_APPLE
```

Comment:
drive by question: how do we decide between `TYSAN_INTERCEPT___STRDUP` and `SANITIZER_APPLE` style conditions?